.github/workflows/linux-pr-tests.yml

```
@@ -97,3 +96,41 @@ jobs:
         with:
           name: logs-linux
           path:  ~/bottles/logs
+
+       - name: Delete logs and home
+         if: always()
+         run: |
+           rm -rvf ~/bottles/logs
+           rm -rvf ~/bottles/home
+
+       - name: Count bottles
+         id: bottles
+         if: always()
+         run: |
+           cd ~/bottles
+           count=$(ls *.json | wc -l | xargs echo -n)
+           echo "$count bottles"
+           echo "::set-output name=count::$count"
+           failures=$(ls failed/*.json | wc -l | xargs echo -n)
+           echo "$failures failed bottles"
+           echo "::set-output name=failures::$failures"
+
+       - name: Upload failed bottles
+         if: always() && steps.bottles.outputs.failures > 0
+         uses: actions/upload-artifact@main
+         with:
+           name: bottles linux
```

```suggestion
           name: bottles-linux
```
I don't think you want spaces in the filename of whatever you end up manually downloading. (Well maybe you do, but I don't 😄)